src/HRQLS/Models/ElasticSearchServiceProvider.php

```diff
@@ -82,12 +84,188 @@ public function boot(Application $app)
     /**
      * Search functionality exposed for use through the service provider.
      *
-     * @param array $req The request to be sent to the ElasticSearch client.
+     * @param array $indices The list of indices to include in the query. Defaults to searching all indices.
+     * @param array $types   The list of types to include in the query. Defaults to searching all types.
+     * @param array $query   The query to run against the specified indexes and types.
```

Comment:
I like the new parameter list.  Kinda jealous I didn't think of something like this.